src/audio3d/Audio3DDevice.h

```diff
@@ -0,0 +1,30 @@
+#ifndef __PIROGRONIAN__SOUND_3D__DEVICE__H__ 
+#define __PIROGRONIAN__SOUND_3D__DEVICE__H__
+
+#include <AL/alc.h>
+
+namespace Audio3D
+{
+    class Device
+    {
+        ALCdevice *_dev;
+
+    public:
+        Device(ALCdevice *dev) : _dev(dev) {}
+        Device() { _dev = alcOpenDevice(NULL); }
+
```

Comment:
remove copy constructor and copy operator

Reply:
I included test code in new commit.